fastfield_codecs/src/column.rs

```diff
@@ -137,37 +137,53 @@ where V: AsRef<[T]> + ?Sized
     }
 }
 
-struct MonotonicMappingColumn<C, T, Input> {
+struct MonotonicMappingColumn<C, T, U, Input> {
     from_column: C,
     monotonic_mapping: T,
+    monotonic_mapping_inv: U,
     _phantom: PhantomData<Input>,
 }
 
 /// Creates a view of a column transformed by a monotonic mapping.
-pub fn monotonic_map_column<C, T, Input: PartialOrd, Output: PartialOrd>(
+/// E.g. apply a gcd monotonic_mapping([100, 200, 300]) == [1, 2, 3]
+/// The provided mappings need to be the inverse of each other.
```

Comment:
That's a tad inaccurate.

`monotonic_mapping` is expected to be injective, and we should always have 
`monotonic_mapping_inv(monotonic_mapping(el)) == el`.

Reply:
(the opposite is not true... Take the GCD mapping for instance.)